src/internal/common/time.go

```diff
@@ -142,14 +146,14 @@ func ParseTime(s string) (time.Time, error) {
 		}
 	}
 
-	return time.Time{}, errors.New("unable to parse time string: " + s)
+	return time.Time{}, clues.Stack(errParsingStringToTime, errors.New(s))
```

Comment:
what does the printed output of this look like? If the string `s` isn't towards the end it could make it harder to quickly parse the logs (as a human that is). I often find it harder to find stuff in the center of strings than the beginning or end

Reply:
Yup!  That was a big driver for the Stack method, actually: not having to push data out of order just to wrap/attach an error.  There's a good display of expectations in the Clues tests, but here it will look like this:
```
parent wrapper message: error parsing string to time: <value of s> 
```